modules/social_features/social_follow_taxonomy/social_follow_taxonomy.install

```diff
@@ -46,3 +61,13 @@ function social_follow_taxonomy_update_8801() {
     \Drupal::service('module_installer')->install(['social_follow_tag']);
   }
 }
+
+/**
+ * Add follow tags filter to "People" page.
+ */
+function social_follow_taxonomy_update_11002(): void {
+  /** @var \Drupal\social_follow_taxonomy\SocialFollowTaxonomyInstallHelper $helper */
+  $helper = \Drupal::classResolver(SocialFollowTaxonomyInstallHelper::class);
+  // Install views filter configuration.
+  $helper->update11002();
```

Comment:
Hi @Ressinel 

I added filter configuration to the method `SocialFollowTaxonomyInstallHelper::getFollowTaxonomyFilterConfiguration()` and made it private to avoid to use it out of the class scope otherwise this class will have "to much power and responsibilities" which is not correct.

I added this `$helper->update11002()` to hook to avoid code duplication of views filter configuration (which exists in the method `SocialFollowTaxonomyInstallHelper::getFollowTaxonomyFilterConfiguration()`. 

I can change the `social_follow_taxonomy_update_11002()` function's body to the below (but we will have the duplications of filter configuration (one in helper, second in hook), is it ok for you?

```php
function social_follow_taxonomy_update_11002(): void {
  // Install views filter configuration.
  $config = \Drupal::configFactory()->getEditable('views.view.user_admin_people');
  $filter = [
    'id' => 'social_follow_taxonomy_follow_filter',
    'table' => 'users_field_data',
    'field' => 'social_follow_taxonomy_follow_filter',
    'relationship' => 'none',
    'group_type' => 'group',
    'admin_label' => '',
    'operator' => 'or',
    'value' => [],
    'group' => 1,
    'exposed' => TRUE,
    'expose' => [
      'operator_id' => 'social_follow_taxonomy_follow_filter_op',
      'label' => 'Following by content tags',
      'use_operator' => FALSE,
      'operator' => 'social_follow_taxonomy_follow_filter_op',
      'operator_limit_selection' => FALSE,
      'operator_list' => [],
      'identifier' => 'social_follow_taxonomy_follow_filter',
      'required' => FALSE,
      'remember' => FALSE,
      'multiple' => TRUE,
      'remember_roles' => [
        'authenticated' => 'authenticated',
      ],
      'reduce' => FALSE,
    ],
    'is_grouped' => FALSE,
    'group_info' => [
      'label' => '',
      'description' => '',
      'identifier' => '',
      'optional' => TRUE,
      'widget' => 'select',
      'multiple' => FALSE,
      'remember' => FALSE,
      'default_group' => 'All',
      'default_group_multiple' => [],
      'group_items' => [],
    ],
    'reduce_duplicates' => FALSE,
    'type' => 'select',
    'limit' => TRUE,
    'vid' => [
      'social_tagging' => 'social_tagging',
    ],
    'hierarchy' => FALSE,
    'error_message' => TRUE,
    'entity_type' => 'user',
    'plugin_id' => 'social_follow_taxonomy_follow_filter',
  ];
  if (!empty($config->get('display.default.display_options.filters'))) {
    $config->set('display.default.display_options.filters.social_follow_taxonomy_follow_filter', $filter);
    $config->save();
  }
}
```

Reply:
Removed helper module and move install login to `social_taxonomy_follow.install `